config/constants.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Constants
+  SEED_ADMIN_EMAIL = ENV.fetch("SEED_ADMIN_EMAIL", nil)
+  SEED_ADMIN_INITIAL_PASSWORD = ENV.fetch("SEED_ADMIN_INITIAL_PASSWORD", nil)
+  MAIL_FROM = ENV.fetch("MAIL_FROM", nil)
+  SIDEKIQ_USERNAME = ENV.fetch("SIDEKIQ_USERNAME", nil)
+  SIDEKIQ_PASSWORD = ENV.fetch("SIDEKIQ_PASSWORD", nil)
```

Comment:
At least those that were mandatory before should still be mandatory

```suggestion
  SIDEKIQ_USERNAME = ENV.fetch("SIDEKIQ_USERNAME")
  SIDEKIQ_PASSWORD = ENV.fetch("SIDEKIQ_PASSWORD")
```